keyboards/gmmk/pro/ansi/keymaps/mattgauf/keymap.c

```diff
@@ -0,0 +1,244 @@
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,               RGB_SPI,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RGB_TOG, _______,          RGB_SAI,      RGB_SPD,
+                      _______, _______, _______,                   _______,              TG(_UTILITY), MO(_DFUMODE), _______,     RGB_HUD, RGB_SAD, RGB_HUI),
+
+  [_UTILITY] = LAYOUT(_______, _______, _______, _______, _______, _______, _______, _______, _______, DM_PLY1, DM_PLY2, DM_REC1, DM_REC2, DM_RSTP,      _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,      _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,      _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,               _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,      _______,
+                      _______, _______, _______,                   _______,                   _______, _______, _______,          _______, _______, _______),
+
+  [_DFUMODE] = LAYOUT(RESET,   _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, MG_F19,       DEBUG,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,      _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,      _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,               _______,
+                      _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,      _______,
+                      _______, _______, _______,                   _______,                   _______, _______, _______,          _______, _______, _______),
+};
+
+
+// Runs just one time when the keyboard initializes.
+void keyboard_post_init_user(void) {
+}
+
+
+// Runs constantly in the background, in a loop.
+void matrix_scan_user(void) {
+}
+
+
+// Called on suspend
+void suspend_power_down_user(void) {
+    rgb_matrix_set_suspend_state(true);
+}
+
+
+// Called on resume
+void suspend_wakeup_init_user(void) {
+    rgb_matrix_set_suspend_state(false);
+}
+
+
```

Comment:
```suggestion

```

Reply:
This block of code is functional on my keyboard. Have I missed something?

Reply:
It shouldn't be needed as of the latest breaking changes.
https://github.com/qmk/qmk_firmware/blob/master/tmk_core/common/avr/suspend.c#L169-L171

Reply:
Gotcha, suggestion committed!